taccsite_cms/static/site_cms/css/src/_imports/trumps/s-inline-dl.css

```diff
@@ -19,6 +19,8 @@ dl.s-inline-dl dt {
   /* Remove space between <dt> and <dd> (from our styles) */
   /* SEE: ../elments/html-elements.html */
   margin-bottom: 0;
+
+  font-weight: var(--bold);
```

Comment:
**Why a Variable?**
This takes longer to write than just `bold`, I know. But the value should be `700`, not `bold`. `700` is not easy to remember.

<details><summary>Edge Case</summary>

__If__ Designer were to change font, the weight chosen for bold could change also, because all fonts' weights do not consistently give the same visual effect.

</details>